src/main/java/com/github/manolo8/darkbot/gui/drawables/TrailDrawer.java

```diff
@@ -70,6 +71,8 @@ private void drawTrail(MapGraphics mg) {
                 last = point;
             }
         }
+
+        if (distance > 50) mg.drawLine(this.last, hero);
```

Comment:
```suggestion
```
I believe this is leftover from an earlier change where line wouldn't always be added